starter-project-web-vue2/components/abel/AbelBlogList.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  <v-row>
+    <v-col v-for="blog in allBlogs" :key="blog.title" cols="12" sm="4">
+      <SingleBlog
+        :title="blog.title"
+        :fullName="blog.authorUserId.fullName"
+        :tag="blog.tag"
+        :content="blog.content"
```

Comment:
Instead of passing individual properties, it is better to pass the object as props